libmamba/src/core/environment.cpp

```diff
@@ -270,6 +272,48 @@ namespace mamba
             return maybe_home;
         }
 
+        fs::u8path user_config_dir()
+        {
+            std::string maybe_user_config_dir = env::get("XDG_CONFIG_HOME").value_or("");
+            if (maybe_user_config_dir.empty())
+            {
+#ifdef _WIN32
+                maybe_user_config_dir = ::mamba::win::get_folder("roamingappdata");
+#else
+                maybe_user_config_dir = home_directory() / ".config";
+#endif
+            }
+            return fs::u8path(maybe_user_config_dir) / "mamba";
```

Comment:
```suggestion
            return fs::u8path(maybe_user_config_dir) / "micromamba";
```

I think it might be better if the folder was called `micromamba`. `mamba` actually uses `conda`'s config resolution and behaves exactly as `conda` in that sense (since it tries to be a 100% drop-in replacement). These things that we specify here only apply to `micromamba` AFAIK.

This might lead to less confusion why `mamba` wouldn't read the "mamba" config files.

WDYT?

Reply:
Hmm, I'd prefer to read those files for Mamba as well and then call it `mamba`, because most options work with Conda, Mamba, Micromamba at the same time and people might use those package managers at the same time.

Reply:
> I'd prefer to read those files for Mamba as well

but is this really the goal with mamba? As far as I understand, mamba should behave like conda in all context. This would include only reading the config files that conda reads.
I doubt conda will start reading ~/.config/mamba…